packages/Liquid-Core.package/LiquidAnswerPanel.class/instance/getChildren.st

```diff
@@ -0,0 +1,5 @@
+accessing
+getChildren
+	^ {self builder build: (self buildMultiChoiceListWith: self builder).
```

Comment:
Is there a semantic difference between a *Multi* Choice question and a *Multiple* Choice one? Could be worth adding to our term definitions

Reply:
I believe you are referencing the naming of `buildMultiChoiceListWith:`? This does not have anything to do with a `MutipleChoiceQuestion`, but refers to the `MultiSelectionList` of the Toolbuilder.

Reply:
I see! Why not name it `buildMultiSelectionListWith:` in that case?

Reply:
Good suggestion. Since we are only using one list in the AnswerPanel at this point, there should be no confusion as to which part of the UI the List refers to.